test/tools/router_check/coverage.cc

```diff
@@ -5,7 +5,7 @@
 namespace Envoy {
 double RouteCoverage::report() {
   uint64_t route_weight = 0;
-  for (const auto& covered_field : coverageFields()) {
+  for (const auto&& covered_field : coverageFields()) {
```

Comment:
This doesn't make sense to me. What was the compiler error you got here?

Reply:
The error from [this job](https://dev.azure.com/cncf/envoy/_build/results?buildId=52843&view=logs&jobId=606a4d51-5ebb-5777-230a-e31fece2034d&j=606a4d51-5ebb-5777-230a-e31fece2034d&t=d7a9df89-dfb4-53e0-7448-280427f79572) was:

```
Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox
test/tools/router_check/coverage.cc:8:20: error: loop variable 'covered_field' is always a copy because the range of type 'std::vector<bool>' does not return a reference [-Werror,-Wrange-loop-bind-reference]
  for (const auto& covered_field : coverageFields()) {
                   ^
test/tools/router_check/coverage.cc:8:8: note: use non-reference type 'std::__1::__bit_reference<std::__1::vector<bool, std::__1::allocator<bool> >, true>'
  for (const auto& covered_field : coverageFields()) {
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
I see. Just make it `auto` or `bool` for the loop variable type to fix it. Thank you!

/wait